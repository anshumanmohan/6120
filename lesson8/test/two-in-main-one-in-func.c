int func(int a, int b)
{
  while (a > b)
  {
    a = a - b;
  }
  return a;
}

int main()
{
  int a = 0;
  for (; a < 100;)
  {
    int b = func(4, 3);
    a = a + b;
  }
  for (; 0 < a; a--)
  {
  }
  return a;
}