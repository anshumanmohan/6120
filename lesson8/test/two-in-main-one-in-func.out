I saw a function called func
	reporting from loop #1
	reporting from loop #1
	the function has 1 loop(s) total
I saw a function called main
	reporting from loop #1
	reporting from loop #1
	reporting from loop #1
	reporting from loop #2
	reporting from loop #2
	the function has 2 loop(s) total
